src/Symfony/Component/Cache/Tests/Adapter/FilesystemTest.php

```diff
@@ -27,4 +27,28 @@ public function createCachePool()
 
         return new FilesystemAdapter('sf-cache');
     }
+
+    public static function tearDownAfterClass()
+    {
+        self::rmdir(sys_get_temp_dir().'/symfony-cache');
+    }
+
+    public static function rmdir($dir)
+    {
+        if (!$dir || 0 !== strpos(dirname($dir), sys_get_temp_dir())) {
+            throw new \Exception(__METHOD__."() operates only on subdirs of system's temp dir");
```

Comment:
According [current docs](http://symfony.com/doc/current/contributing/code/standards.html#structure), the exception message should be concatenated with `sprintf()` using single quotes:
>Exception and error message strings should be concatenated using sprintf